cityofaustin/atd-vz-data
# Remove unused components from VZE

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17392

Ended up finding a lot more unused components than just the Charts

## Testing
**URL to test:** 
Locally

**Steps to test:**
1. Check out this branch and start your data model DB
2. Check around the app to make sure nothing is broken
3. Admire all the red :heart_eyes: 

---
#### Ship list
